packages/extension-ui/src/components/Button.tsx

```diff
@@ -94,7 +97,9 @@ export default styled(Button)(({ isDanger, theme }: Props) => `
     background: rgba(96,96,96,0.15);
 
     .children {
-      opacity: 0.25;
+      /* opacity: 0.25; */
```

Comment:
If you comment something out please write a over or next to line why it's commented out with 'todo' like:
```
// todo: temporary - remove or enable after xyz
```